packages/editor/src/components/post-author/index.js

```diff
@@ -112,4 +153,4 @@ function PostAuthor() {
 	);
 }
 
-export default PostAuthor;
+export default compose( [ withInstanceId ] )( PostAuthor );
```

Comment:
we have a hook now `useInstanceIdd` to replace the HoC

Reply:
👍🏼